tests/test_github_forwarders_logic.py

```
@@ -150,6 +151,156 @@ def log_message(self, format: str, *args: object) -> None:  # noqa: A003
+                    body = json.dumps(payload).encode("utf-8")
+                    self.send_response(200)
+                    self.send_header("Content-Type", "application/json")
+                    self.send_header("Content-Length", str(len(body)))
+                    self.end_headers()
+                    self.wfile.write(body)
+                    return
+
+                self.send_response(404)
+                self.end_headers()
+
+            def log_message(self, format: str, *args: object) -> None:  # noqa: A003
```

Ruff (0.15.9)

[error] Function argument `format` is shadowing a Python builtin

(A002)